server/app/routes/orders/index.js

```diff
@@ -16,6 +16,16 @@ router.get('/', (req, res, next) => {
 	.catch(next);
 });
 
+router.get('/user/:userId', (req, res, next) => {
```

Comment:
Restfully this would just be /:userId I think - I don't think you need the first /user/